src/site/content/en/blog/layoutNG/index.md

```diff
@@ -0,0 +1,234 @@
+
+#### Orthogonal Flows
+LayoutNG improves vertical flow layout correctness, for more correct positioning of absolute positioned objects, sizing of orthogonal flow boxes especially when percent is used, and so forth. Among the 1,258 tests in W3C test suites, **103 tests that failed in the old engine pass in LayoutNG.**
+
+#### Intrinsic Sizing
+Intrinsic sizes are now calculated correctly when a LayoutNG block contains children in an orthogonal writing mode.
+
+<br>
+
+### Text Layout & Line Breaking
+The old layout engine in Chromium does text layout element-by-element and line-by-line. This has historically worked well but requires a lot of extra complexity to support scripts and to achieve good performance. It's also prone to inconsistencies in measurements which tend to manifest themselves as subtle differences in sizing of size-to-content containers and their content or unnecessary line breaks.
+
+In LayoutNG, text layout is done on a paragraph level and is then split into lines. This allows for better performance, higher quality text rendering, and more consistent line breaking. The most notable differences here from a developer and user standpoint are detailed below.
+
+#### Joining across Element Boundaries
+In some scripts graphemes join with adjacent ones and changes presentation. In LayoutNG this works even if the graphemes are in different elements, allowing the joins to be preserved even if different styling is applied ([#6122](https://crbug.com/6122)).
+
+{% Aside 'key-term' %}
+  a grapheme is the smallest unit of a writing system of any given language
+{% endAside %}
+
+The example below shows the rendering of the following HTML in legacy layout and LayoutNG respectively:
+
+```html
+<div>&#1606;&#1587;&#1602;</div>
+<div>&#1606;&#1587;<span>&#1602;</span></div>
+```
+
+<div class="w-figures-2-up">
+  <figure class="w-figure w-figure--center">
+    <img loading="lazy" src="legacy_shape.png" alt="proper grapheme on left and separated improper rendering on right">
+    <figcaption class="w-figcaption w-figcaption--center">
+      <small>Fig 3a, Legacy Layout</small><br>
+      Note how the form of the second letter changes
+    </figcaption>
+  </figure>
+
+  <figure class="w-figure w-figure--center">
+    <img loading="lazy" src="ng_shape.png" alt="proper combined graphemes shown">
+    <figcaption class="w-figcaption w-figcaption--center">
+      <small>Fig 3b, LayoutNG</small><br>
+      Note they're now identical
+    </figcaption>
+  </figure>
+</div>
+
+#### CJK Ligatures
+Although Chromium already supports ligatures and enable them by default, there are some limitations. Ligatures involving multiple CJK codepoints are not supported in the legacy engine due to a rendering optimization. LayoutNG removes these restrictions and supports ligatures regardless of script.
+
+The example below shows the rendering of three discretionary ligatures using the Adobe SourceHanSansJP font:
```

Comment:
```suggestion
The example below shows the rendering of three discretionary ligatures using the Adobe SourceHanSansJP font:
```
Define "discretionary"? Or omit if possible? (I'm mindful of how the jargon related to text rendering is building up in this section...)

Reply:
hm, no opinions here. good question for original author maybe?